.gitignore

```diff
@@ -4,3 +4,9 @@ local.properties
 # Gradle
 build
 .gradle
+
+# Android Studio
+.idea/*
```

Comment:
We tend to omit IDE-related stuff in project `.gitignore`. Number of IDEs or editors people can use to edit code is not limited. Hence, theoretically we cannot cover all the cases here. `.gitignore` rules in this files are related to the build system used in the project. And IDE-related stuff is recommended to be kept in [global git ignore configuration](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) on your machine to fit your preferences and avoid copying it in all the projects.